exporters/metric/cortex/cortex_test.go

```diff
@@ -0,0 +1,19 @@
+package cortex_test
+
+import (
+	"testing"
+
+	"go.opentelemetry.io/otel/sdk/export/metric"
+	"go.opentelemetry.io/otel/sdk/export/metric/aggregation"
+	"opentelemetry.io/contrib/exporters/metric/cortex"
+)
+
+func TestExportKindFor(t *testing.T) {
```

Comment:
Seems kind of silly to have a test for a one line function, but I've heard OT has high requirements on test coverage for contributions, so I guess it makes sense